### PR TITLE
Limit slider commands

### PR DESCRIPTION
**Pull request by bvanwijk**
The focus, zoom and iris inputs can now only send commands to the backend once in 130 ms.

### CODE REVIEW

**Comment by jochem725** [comment]
Seems fine however I think we need some security for this on the backend server to guarantee a code change in the future will not flood a camera.

**Comment by bvanwijk** [comment]
@jochem725 I think that is part of another task on the sprint of this week.

File: frontend/public/js/control.js [hunk 1]
```diff
@@ -158,36 +159,91 @@ function sendMove(){
...
 	$('#auto_focus').removeClass("btn-success");
-	$.get("/api/backend/camera/" + currentcamera + "/focus?autoFocusOn=false&position=" + focus , function(data) {});
-	console.log("Focus: " + focus);
+	if (focusSend === false) {
+		focus = f;
+		focusSend = true;
+		setTimeout(function(){sendFocus(); focusSend = false;}, 130);
+		sendFocus();
+	} else {
+		focus = (parseInt(f) + parseInt(focus)) / 2;
```
Comment (Kingdorian) [inline frontend/public/js/control.js]: I see this code (strucure) 3 times in the code. Maybe you can turn this in to a more generalized method to get rid of the duplicate code.

Comment (bvanwijk) [inline frontend/public/js/control.js]: @Kingdorian Yes, I agree this is duplicate code. I changed it, do you agree with merging now?

**Comment by Kingdorian** [comment]
Great I think this can be merged now!

**Pull request merged**